yt_dlp/extractor/mediaset.py

```diff
@@ -29,7 +28,7 @@ class MediasetIE(ThePlatformBaseIE):
                     '''
     _TESTS = [{
         # full episode
-        'url': 'https://www.mediasetplay.mediaset.it/video/mrwronglezionidamore/episodio-1_F310575103000102',
+        'url': 'https://mediasetinfinity.mediaset.it/video/mrwronglezionidamore/episodio-1_F310575103000102',
```

Comment:
Leave one of the tests in old domain

Reply:
can I make a duplicate and make it "only_matching"?